src/styles/bloks/calendar__day.scss

```diff
@@ -0,0 +1,45 @@
+.calendar__day {
```

Comment:
`calendar__day` is a bem element of `calendar`, no need to create a separate file for this